app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from './AdminEmail.module.scss';
+import CopyToClipboard from '../../../../Components/CopyToClipboard/CopyToClipboard';
+import { Button } from 'kwc';
+import IconSend from '@material-ui/icons/Send';
+import React from 'react';
+
+type Props = {
+  email: string;
+};
+function AdminEmail({ email }: Props) {
+  function onSendEmail() {
```

Comment:
this also should be named: `handleSendEmail`:
```suggestion
  function handleSendEmail() {
```